data/human/human missions.txt

```diff
@@ -3996,3 +3996,132 @@ mission "Fumbling Mugger"
 event "fumbling mugger followup timer"
 
 
+mission "Riley"
+	name "Visiting Home"
+	description "Take a woman named Riley to see her ailing father on <destination> before it's too late."
+	minor
+	passengers 1
+	source
+		attributes deep 
+		attributes urban
+	deadline
+	destination "Pilot"
+	to offer
+		random < 15
+	on offer
+		conversation
+			`Entering the spaceport a young woman is moving from captain to captain, looking for transport.`
+			choice
+				`	(Ask the young woman where she needs to get to).`
+				`	(Ignore her).`
+			`	"I need to get to <destination> to see my father. He's dying. I have <payment> credits to pay for the journey; please, will you help me? I need to be there in <deadline> days, at the most."`
+			`	Her eyelids are baggy, her clothes wrinkled, and she holds herself rigidly; signs of desperation, stress, and sleep deprivation.`
+			choice
+				`	"I can help you. We'll make haste to reach <planet> in time."`
+					goto help
+				`	"With that much money, why not buy your own ship?"`
```

Comment:
You can't buy a ship with 13,000 credits (if I read the payment right)

Reply:
I upped it to the cost of a shuttle, but I feel like that's a bit excessive for a transport job. I'm thinking of just removing that line entirely.

Reply:
Yeah, I'm guessing that Starsector has EV-style prices, where a shuttle is 10-20k credits. That'll be real tough to draw a comparison in ES for.